codalab/apps/common/utils.py

```diff
@@ -1,15 +1,18 @@
-def recent_worksheets(worksheets):
-    """Function to return most recent worksheets"""
+import random
 
-    if not worksheets:
-    	return worksheets  # just incase the list is empty
+from django.conf import settings
 
-    sorted_worksheets = sorted(worksheets, key=lambda k: k['id'], reverse=True)
 
-    if len(sorted_worksheets) <= 2:
-        worksheets = [(val['uuid'], val['name'], val['owner_name'])for val in sorted_worksheets]
-        return worksheets
+def random_worksheets(list_worksheets):
```

Comment:
Getting the list of all worksheets and then filtering is expensive.  Once we have worksheet tags, we should be using search_worksheets to do that.  For now, put all the logic (getting the list of worksheets and filtering) in one place (here) so we can change it later easily.